packages/dashboard/src/components/notification-bar.tsx

```diff
@@ -1,34 +0,0 @@
-import { Snackbar } from '@material-ui/core';
```

Comment:
This may still be useful, let's keep the code but leave it unused. Let's also rename it to `AlertSnackBar` or something similar to avoid confusion with the new notification dialog.